app/src/main/java/com/example/sharingang/SetPriceActivity.kt

```diff
@@ -0,0 +1,32 @@
+package com.example.sharingang
+
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.view.View
+import android.widget.EditText
+
+class SetPriceActivity : AppCompatActivity() {
+    companion object {
+        const val EXTRA_PRICE = "com.example.sharingang.EXTRA_PRICE"
+        var CHOSEN_CURRENCY = "USD"
+    }
+
+    private val defaultPrice = 0.0
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_set_price)
+    }
+
+    fun setPriceSummary(view: View) {
+        val intent = Intent(this, SetPriceSummaryActivity::class.java)
+        val editTextPrice : EditText = findViewById(R.id.editTextSetPrice)
+        val price =
+            if(editTextPrice.text.isNullOrBlank()) defaultPrice // if no price has been entered
```

Comment:
I think you've forgotten to format this file, because there would likely be a space between the if and the parens here, and there would be no space between `editTextPrice` and `:`